core/src/model/Entity.java

```diff
@@ -89,23 +91,37 @@ public float getMaxHealth() {
         return maxHealth;
     }
 
-    private void setMaxHealth(float maxHealth) {
+    protected void setMaxHealth(float maxHealth) {
         this.maxHealth = maxHealth;
     }
 
     public float getCurrentHealth() {return currentHealth;}
 
     public void setCurrentHealth(float currentHealth) {this.currentHealth = currentHealth;}
 
+    public float getDamage() {
+        return damage;
+    }
+    protected void setDamage(float damage) {
+        this.damage = damage;
+    }
+
     public Direction getDirection() {
         return direction;
     }
 
-    private void setDirection(Direction direction) {
+    protected void setDirection(Direction direction) {
         this.direction = direction;
     }
 
-    private World<Entity> getWorld() {
+    public boolean getInMotion() {
```

Comment:
The speed in Entity is a constant. Is it ever set to 0?

Reply:
I dont understand ur question